### DISCUSSION TITLE
Hiding/Showing columns AnalyticalTable

### DISCUSSION THREAD

**Discussion by Blerina** [UI5/webcomponents-react#1886] [Q&A] [answered]
Hello, 

I was trying to find a way to show or hide columns on demand for an AnalyticalTable component. 
Is there a plan to add this feature as part of the AnalyticalTable so users can select which columns to display or hide? 
What alternatives we could have with the existing component?

Thank you very much in advance!

**Comment by MarcusNotheis** [comment, marked as answer]
Hey @Blerina,
I've added my answer in #1899. Can I mark this discussion as answered then?

**Reply by Blerina** [reply]
Thank you @MarcusNotheis! Yes, we can mark this as answered (I tried to delete it but I wasn't able to find this option).
